src/pudl_archiver/metadata/sources.py

```diff
@@ -44,4 +145,121 @@
+            "in a variety of tabular and geospatial file formats, to meet a range of"
+            "user/software needs. Dynamic web services are available for users that wish"
+            "to access the USWTDB as a Representational State Transfer Services (RESTful)"
+            "web service."
+        ),
+        "working_partitions": {
+            "year_months": [
+                "2018-04",
+                "2018-07",
+                "2018-10",
+                "2019-01",
+                "2019-04",
+                "2019-07",
+                "2019-10",
+                "2020-01",
+                "2020-04",
+                "2020-05",
+                "2020-07",
+                "2020-10",
+                "2021-01",
+                "2021-04",
+                "2021-07",
+                "2021-11",
+                "2022-01",
+                "2022-04",
+                "2022-07",
+                "2022-10",
+                "2023-01",
+                "2023-05",
+                "2023-11",
+                "2024-05",
+                "2024-11",
+            ]  # these are almost year_quarters but not quite....
+        },
+        "keywords": sorted(
+            {
+                "usgs",
+                "uswtdb",
+                "wind",
+                "wind turbines",
+            }
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "usgsuspvdb": {
+        "title": "USGS USPVDB -- U.S. Large-Scale Solar Photovoltaic Database",
+        "path": "https://energy.usgs.gov/uspvdb/",
+        "description": (
+            "The United States Large-Scale Solar Photovoltaic Database (USPVDB) provides"
+            "the locations and array boundaries of U.S. ground-mounted photovoltaic (PV)"
+            "facilities with capacity of 1 megawatt or more. It includes corresponding PV"
+            "facility information, including panel type, site type, and initial year of"
+            "operation. The creation of this database was jointly funded by the U.S."
+            "Department of Energy (DOE) Solar Energy Technologies Office (SETO) via the"
+            "Lawrence Berkeley National Laboratory (LBNL) Energy Markets and Policy"
+            "Department, and the U.S. Geological Survey (USGS) Energy Resources Program."
+            "The PV facility records are collected from the U.S. Energy Information"
+            "Administration (EIA), position-verified and digitized from aerial imagery,"
+            "and checked for quality. EIA facility data are supplemented with additional"
+            "attributes obtained from public sources."
+        ),
+        "working_partitions": {"years": [2023, 2024]},
+        "keywords": sorted(
+            {
+                "usgs",
+                "uspvdb",
+                "solar",
+                "pv",
+                "photovoltaic",
```

Comment:
Maybe include the other funders
```suggestion
                "photovoltaic",
                "doe",
                "seto",
                "lbnl",
```